src/app/center-bottom-content/components-bottom/report/report-builder/report-builder.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, Input, OnInit, Output, EventEmitter } from '@angular/core';
+import { Workflow } from 'src/app/shared/interfaces/workflow';
+import { AppService } from 'src/app/shared/services/app.service';
+
+@Component({
+  selector: 'app-report-builder',
+  templateUrl: './report-builder.component.html',
+  styleUrls: ['./report-builder.component.scss']
+})
+export class ReportBuilderComponent implements OnInit {
+  // show/hide for report builder compenent and workflow component
+  @Input() showHideReportBuilder!: boolean;
+  @Output() isReportBuilder = new EventEmitter<boolean>();
+  @Input() showHideWorkflow!: boolean;
+  @Output() isWorkflow = new EventEmitter<boolean>();
+
+  // component communication between report builder and center bottom
+  @Input() workflows: Array<Workflow> = [];
+  @Output() selectedWorkflows = new EventEmitter<any>();
```

Comment:
What I usually use Inputs and Outputs for is sharing data that you want to specifically go to one place, but not other places. I talk about this below when I mention moving the workflows inside the report-builder, and sending an input to tell the workflow component which workflow that instance will represent.

For a lot of the things set up in this section, I don't think we need inputs and outputs to store those variables, since there's nothing that we want to _only_ go into a specific version of a component. All of these items can be available to the whole app through subjects and observables in a service, or can just be done in the report builder component itself (rather than the bottom bar component).